external-crates/move/crates/move-package/src/resolution/resolution_graph.rs

```diff
@@ -441,6 +455,29 @@ impl Package {
             .collect())
     }
 
+    fn get_build_paths(package_path: &Path) -> Result<Vec<PathBuf>> {
+        let mut places_to_look = Vec::new();
+        let path = package_path.join(Path::new(DEFAULT_OUTPUT_DIR));
+        if path.exists() {
+            places_to_look.push(path);
+        }
+        Ok(places_to_look)
+    }
+
+    pub fn get_bytecodes(&self) -> Result<Vec<FileName>> {
+        let path = Package::get_build_paths(&self.package_path)?;
+        let places_to_look = path
+            .into_iter()
+            .map(|p| p.to_string_lossy().to_string())
```

Comment:
This pattern is not great -- especially because we are using the resulting `places_to_look` as file paths.  Is there a way to change `find_filenames` to accept paths, to avoid doing this lossy conversion?

Reply:
This was trivial as `find_filenames` already accepts `&[impl AsRef<path>]` for the paths argument